src/Utils/Timer.h

```diff
@@ -0,0 +1,23 @@
+//
+// Created by DANIELA CARRERO on 2020-10-26.
+//
+
+#ifndef TPTALLER1_TIMER_H
+#define TPTALLER1_TIMER_H
+
+
+#include <SDL.h>
```

Comment:
If this is okay with you, please change this to be SDL2/SDL.h

Reply:
También lo cambié